packages/core/src/semanticAttributes.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Use this attribute to represent the source of a span.
+ * Should be one of: custom, url, route, view, component, task, unknown
+ *
+ */
+export const SEMANTIC_ATTRIBUTE_SENTRY_SOURCE = 'sentry.source';
```

Comment:
might be a good thing to put under a subpath export `@sentry/core/attributes`, let's give it a try in v8.